src/parser/shared/metrics/apl/conditions/debuffMissing.tsx

```diff
@@ -73,7 +76,7 @@ export function debuffMissing(
       const targets = getTargets(event, targetOptions?.targetLinkRelation);
       if (targets.length === 0) {
         //No target so we can't check for a debuff
-        return false;
+        return fallback;
```

Comment:
hmm, i wonder if `fallback = true` should be the default. that would be a breaking change so *not now* but...maybe